com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -413,36 +414,6 @@ void GenerateShaderPass(int targetIndex, PassDescriptor pass, ActiveFields activ
                 spliceCommands.Add("PassPragmas", command);
             }
 
```

Comment:
These move further down, so we can combine the Target/pass includes with the node includes before generating them.